modules/configuring-scale-bounds-knative.adoc

```diff
@@ -16,6 +16,10 @@ maxScale:: If the `maxScale` annotation is not set, there will be no upper limit
 
 [source,yaml]
 ----
+ApiVersion: serving.knative.dev/v1
```

Comment:
I'd not adding this here. This was a YAML fragment. By adding this it seems like this is fully working example, but it isn't.